File: 0252_Meeting Rooms/MeetingRooms.cs
public class Solution {
    public bool CanAttendMeetings(int[][] intervals) {
        Array.Sort(intervals, (a,b) => { return a[0].CompareTo(b[0]); });
        for(int i=1;i<intervals.Length;i++)
            if(intervals[i][0] < intervals[i-1][1]) return false;
        return true;
    }
}

File: 0252_Meeting Rooms/MeetingRooms.java
class Solution {
    public boolean canAttendMeetings(int[][] intervals) {
        Arrays.sort(intervals, (a,b) -> {
            return Integer.compare(a[0],b[0]);
        });

        for(int i=1;i<intervals.length;i++) {
            if(intervals[i][0] < intervals[i-1][1]) return false;
        }

        return true;
    }
}